### PR TITLE
Sponsor

### PR DESCRIPTION
**Pull request by Sibaim**

### CODE REVIEW

File: app/models/sponsor.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class Sponsor < ApplicationRecord
+
+validates :company_name, presence: true, length: {maximum: 50}
```
Comment (arnoFleming) [inline app/models/sponsor.rb]: please reindent

Comment (arnoFleming) [inline app/models/sponsor.rb]: Why a max of 50 chars?

File: spec/models/sponsor_spec.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe Sponsor, type: :model do
+
```
Comment (arnoFleming) [inline spec/models/sponsor_spec.rb]: please remove extraneous newlines

File: spec/models/sponsor_spec.rb [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+
+      describe "validations" do
```
Comment (arnoFleming) [inline spec/models/sponsor_spec.rb]: please reindent

File: spec/factories/sponsors.rb [hunk 4]
```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
```
Comment (arnoFleming) [inline spec/factories/sponsors.rb]: you don't use this factory. consider removing it, or using it in your specs

File: spec/models/sponsor_spec.rb [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+          sponsor.valid?
+          expect(sponsor.errors).to have_key(:company_name)
+        end
+
+        it "is invalid with a Company name longer than 50 characters" do
+          sponsor = Sponsor.new(company_name: "Lorem ipsum dolor sit amet, consectetuer adipiscing elit. ")
+          sponsor.valid?
+          expect(sponsor.errors).to have_key(:company_name)
+        end
+      end
```
Comment (arnoFleming) [inline spec/models/sponsor_spec.rb]: please add a test for a valid sponsor

**Comment by JTBreunissen** [comment]
lgtm

**Pull request closed without merging**